docs/msf/msf-api-guide.mdx

```diff
@@ -16,15 +16,15 @@ Previously, a store had a collection of categories that were organized in a tree
 
 To access the category structure used by a particular storefront, you must first identify which tree is tied to that storefront channel. You can do so by sending a request to the following endpoint. Include the `channel_id:in` query parameter, which accepts one or more channel IDs as a comma-separated list. 
 
-```http title="Example request: Get category tree for channel 3"
+```http filename="Example request: Get category tree for channel 3" showLineNumbers
```

Comment:
```suggestion
```http filename="Example request: Get category tree for channel 3" showLineNumbers copy
```